pkg/parser/parameters.go

```diff
@@ -449,6 +449,13 @@ func (doc *YamlDocument) parseSimpleParameterValue(paramName string, simpleParam
 		return ast.ParameterValue{}, fmt.Errorf("error while parsing simple parameter value")
 	}
 
+	// This is needed if a parameter is written such as :
+	//     param: >
+	//       value
+	if simpleParamNodeChild.Type() == ">" {
```

Comment:
The multiline syntax work also with `|`.

```suggestion
	if simpleParamNodeChild.Type() == ">" || simpleParamNodeChild.Type() == "|" {
```